### ISSUE TITLE
Windows 10 HAMX required fail

### ISSUE THREAD

**Issue by krotofil** [intel/haxm#482]
hello guys, i just will installing haxm 7.7.1 on my Workstation, but i getting an error:

System Requirements:
Intel(R) Core(TM) i7-7800X CPU @ 3.50GHz   3.50 GHz
64-Bit, x64-CPU
SystemBuild Windows Enterprise 10 19042.2251

Check Bios Settings:
Virtualization Technology VTx - checked
Virtualization Technology for Directed I/O VTd - checked

Issue:

Version: 7.7.1
Output folder: C:\Program Files\Intel\HAXM
Extract: checktool.exe... 100%
Execute: C:\Program Files\Intel\HAXM\checktool.exe --verbose
CPU vendor          *  GenuineIntel
Intel64 supported   *  Yes
VMX supported       -  No
VMX enabled         -  No
EPT supported       -  No
NX supported        *  Yes
NX enabled          *  Yes
Hyper-V disabled    -  No
OS version          *  Windows 10.0.19042
OS architecture     *  x86_64
Guest unoccupied    *  Yes. 0 guest(s

**Comment by wayne-ma** [comment]
Please disable Hyper-V then try again. Or try the new version of HAXM v7.8.0.

**Comment by soroshsabz** [comment]
I install the latest HAXM v7.8.0, and my CPU is 12900KS, but when I run 

```
C:\Users\sooro>"C:\Program Files\Intel\HAXM\checktool.exe"
CPU vendor          *  GenuineIntel
Intel64 supported   *  Yes
VMX supported       -  No
VMX enabled         -  No
EPT supported       -  No
NX supported        *  Yes
NX enabled          *  Yes
Hyper-V disabled    -  No
OS version          *  Windows 10.0.22621
OS architecture     *  x86_64
Guest unoccupied    *  Yes. 0 guest(s)
```

see VMX supported No, and Hyper-V disabled No, but I disable HyperV and off launchtype with `bcedit`

**Comment by wcwang** [comment]
Thanks for your feedback. You may follow the [wiki page](https://github.com/intel/haxm/wiki/Windows-System-Configurations) to configure your Windows host and then install HAXM [v7.7.1](https://github.com/intel/haxm/releases/tag/v7.7.1).

**Comment by soroshsabz** [comment]
@wcwang exactly, I try to configure my system based on your wiki page, but my problem does not resolved. I think new Windows Kernel does not work correctly with HAXM, as you can see in [Hardware acceleration for emulator performance (Hyper-V & HAXM)](https://learn.microsoft.com/en-us/xamarin/android/get-started/installation/android-emulator/hardware-acceleration?tabs=vswin&pivots=windows#hyper-v-win)

_On Windows 10 October 2018 Update (RS5) and higher, you only need to enable Hyper-V, as it will use Windows Hypervisor Platform (WHPX) automatically._

**Comment by soroshsabz** [comment]
The other note is, why VMX and EPT says ***NO*** ?

**Comment by wcwang** [comment]
Thanks for your reply. If Hyper-V is on, the CPUID instruction cannot read real value of VMX and EPT. Have you ever checked the Sandbox status described in wiki page?
